SDK/AppCenterData/Microsoft.AppCenter.Data.Shared/Data.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.AppCenter.Data
+{
+    public partial class Data<T>
```

Comment:
Generics are by method, not on the class. You can work with different document types but there is only 1 instance.

Reply:
This empty file is in the wrong project. Please study how we did the shared pattern in other modules (all except analytics).